core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/message.jelly

```diff
@@ -0,0 +1,10 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+    <div class="warning">
+        ${%warningMessage}
+        <a href="https://jenkins.io/redirect/csrf-protection" target="_blank">${%this page}</a>.
```

Comment:
This seems like a hackish combination of localized strings (not every language may be able to finish the sentence with what makes sense as a link label). Would be preferable to have the link inside the localized string.

Reply:
It would require wrapping the localized string with `j:out` IIUC. I am not sure I am ready to advertise it

Reply:
It won't. There's a specific behavior implemented in Stapler Jelly to not escape HTML from localized strings. So just move the entire `div` content in there, and done.

Reply:
corrected as requested, thanks for the information about trusted html in properties